### ISSUE TITLE
Do you still work on this app ?

### ISSUE THREAD

**Issue by zoroya1** [T8RIN/Firenote#2]
Looks nice, I was wondering if we can expect new features or versions

**Comment by T8RIN** [comment]
Yes, maybe in the future i will continue maintenance, because i almost done everything in ImageToolbox

**Comment by zoroya1** [comment]
Thank you for the cheerful info...closing this as completed ✨

**Comment by T8RIN** [comment]
Let it be opened 👀
